sdk/java/pulumi/src/main/java/com/pulumi/resources/Resources.java

```diff
@@ -45,4 +45,8 @@ protected static <T extends Copyable<T>> T copyNullable(@Nullable T original) {
         }
         return original.copy();
     }
+
+    protected static <T> List<T> orEmpty(@Nullable List<T> list) {
```

Comment:
Nit: this is a bit weird to put it here (though protected); I'd perhaps just inline this.

Reply:
I agree, esp. because it is protected, I'd just inline. A little repetition is better than a ton of dependency in my book ;)